postings/urls.py

```diff
@@ -1,9 +1,8 @@
 from django.urls import path
 
-from .views import PostingDeatilView, CommentView
+from .views import PostingDeatilView, PostingLikeView
 
 urlpatterns=[
     path('/<int:posting_id>', PostingDeatilView.as_view()),
-    path('/<int:posting_id>/comments', CommentView.as_view()),
-    path('/<int:posting_id>/comments/<int:comment_id>', CommentView.as_view())
+    path('like/<int:posting_id>/', PostingLikeView.as_view())
```

Comment:
```suggestion
    path('/like', PostingLikeView.as_view())
```
위와 같이 수정하면 더 좋을 것 같습니다.

Reply:
넵 수정 완료 했습니다~